autogpt/core/__init__.py

```diff
@@ -1,11 +1,77 @@
-from autogpt.core.agent import Agent
-from autogpt.core.budget import BudgetManager
-from autogpt.core.command import Command, CommandRegistry
-from autogpt.core.configuration import Configuration
-from autogpt.core.llm import LanguageModel
-from autogpt.core.logging import Logger
-from autogpt.core.memory import MemoryBackend
-from autogpt.core.messaging import Message, MessageBroker
-from autogpt.core.planning import Planner
-from autogpt.core.plugin import Plugin, PluginManager
-from autogpt.core.workspace import Workspace
+import click
```

Comment:
Move this out of here and into something like `autogpt.cli.v2`.  `core` will operate like a library interface, so this init will be non-trivial as we get to implementation.  Alternatively we could have the cli and the app sketch live in `autogpt.re_arch_app` or similar.  Let me know what you think as I'll have to do some motion in other branches.